docs/docs/rasa-pro-release-notes.mdx

```diff
@@ -0,0 +1,61 @@
+---
+id: rasa-pro-release-notes
+sidebar_label: Rasa Pro Release Notes
+title: Rasa Pro Release Notes
+---
+
+## Rasa Pro 3.6
+
+### Rasa Plus 3.6
+
+#### Features
+- Something about PII
+- Something about Analytics Markers
```

Comment:
this can be used: https://github.com/RasaHQ/rasa-plus/blob/3.6.x/changelog/685.feature.md
and/or this: https://github.com/RasaHQ/rasa-pro-services/blob/main/changelog/151.feature.md